### PR TITLE
exposing dashboards for hypertrace metrics

### PR DESCRIPTION
**Pull request by kotharironak**
## Description
As part of making monitoring of hypertrace easy as described in this issue (https://github.com/hypertrace/hypertrace-ingester/issues/109), exposing dashboards for metrics exposed via /metrics endpoints of hypertrace services.

### Testing
Followed below steps,
- starting locally running hypertrace service using docker-compose
- running prometheus locally
- exploring dashboards to locally running grafana instance

### Checklist:
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged and published in downstream modules

### Documentation
We will update the docs for the above metrics

### CODE REVIEW

**Review by rish691 — COMMENTED** [review]
Why the serial number prefix in the file names? 
001-ingestion-pipeline-metrics.json -> ingestion-pipeline-metrics.json

**Comment by kotharironak** [comment]
> Why the serial number prefix in the file names?
> 001-ingestion-pipeline-metrics.json -> ingestion-pipeline-metrics.json

Gave each dashboard a number - kind of <id-name> pattern. I have seen this pattern on `kafka-exporter` side.

**Review by rish691 — COMMENTED** [review]
I think we should update the readme with a section on Monitoring, with information regarding:
- How to setup monitoring and use these json files
- Reference to meaning of metrics

**Review by rish691 — APPROVED** [review]
LGTM

File: kubernetes/monitoring/README.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+## Monitoring Hypertrace
+To help in monitoring hypetrace, it exposes a bunch of application and resource level metrics by using 
+certain libraries - micrometer (https://github.com/micrometer-metrics/micrometer) + DropWizard 
+(https://github.com/dropwizard/metrics). It is also configured to report all of the metrics to Prometheus 
+as default reporter. Each pod of the `hypertrace-service` and  `hypertrace-ingester` are annotated 
+with `prometheus.io/scrape: true`, so by deploying Prometheus-operator (https://prometheus-community.github.io/helm-charts/) 
+in your kube environment, you can have access to all the metrics.
```
Comment (rish691) [inline kubernetes/monitoring/README.md:7]: Could we also add instruction to run the prometheus operator and grafana along with ht in k8s.
Maybe a script can be added to do that

Comment (JBAhire) [reply kubernetes/monitoring/README.md]: I won't care a lot about that, as it's a part of standard tooling at most of the companies. It's just deploying an operator in same cluster. I have linked to prometheus operator github repository now which should give detailed instructions about installation.

Comment (rish691) [reply kubernetes/monitoring/README.md]: My point was, should we provide a script to setup monitoring for HT

**Review by JBAhire — APPROVED** [review]
LGTM

**Review by rish691 — APPROVED** [review]

**Pull request merged**